lib/kvdb/kvdb_ctxn.c

```diff
@@ -99,6 +114,9 @@ kvdb_ctxn_unlock_impl(struct kvdb_ctxn_impl *ctxn)
 {
     mutex_unlock(&ctxn->ctxn_lock);
 }
+// Remove "_impl" from ctx lock functions?
+// Maybe even remove "kvdb_" since they are static?
```

Comment:
Greg seems to be on a kick of including the prefix for static functions that relate to the object at hand. I don't necessarily disagree with his statement.

Reply:
It keeps symbols sorted by name, so when you're looking at object dumps, stack traces, and live function sampling you can easily see that the functions belong to some family of related functions, vs a function name from left field that tells you nothing...

Reply:
I didn't notice that `kvdb_ctxn_trylock_write` makes use of `kvdb_ctxn_trylock_impl`.  The `_impl` make more sense to me now.